src/anbox/cli.h

```diff
@@ -243,6 +243,9 @@ class Command : public DoNotCopyOrMove {
   /// @brief description returns a longer string explaining the command.
   virtual Description description() const;
 
+  /// @brief hidden returns if the command is hidden from the user or not.
+  virtual bool hidden() const;
```

Comment:
Let me rework that in a follow up PR. Want to get this out today and I am lazy :-)